test/exercism/use_cases/attempt_test.rb

```diff
@@ -149,8 +150,8 @@ def test_previous_submission_after_superseding
   end
 
   def test_previous_submission_with_new_language_sandwich
-    Attempt.new(user, 'CODE 1', 'two/two.rb').save
-    Attempt.new(user, 'CODE 2', 'two/two.py').save
+    Attempt.new(user, Iteration.new({'two/two.rb' => 'CODE 1'}, {track: 'ruby', slug: 'two'})).save
```

Comment:
Line is too long. [99/80]<br>Redundant curly braces around a hash parameter.<br>Space inside { missing.<br>Space inside } missing.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.